docs/data/variant-calls.md

```diff
@@ -16,6 +16,16 @@ Data files containing Single Nucleotide Variations (SNV) called from aligned rea
 | ---------------- | ------------------------------- | --------------- | --------------------------- | ----------------------------------------------------------------------------------------------------------------------- |
 | \*.snv.vcf.gz    | SNV results in VCF File format. | variant_calling | Simple Nucleotide Variation | <ul><li>Sanger WGS Variant Calling </li><li>Sanger WXS Variant Calling </li><li> GATK Mutect2 Variant Calling</li></ul> |
 
+## Filtered SNV Calls
+
+Data files containing Single Nucleotide Variations (SNV) variants which were filtered through each caller's PASS tag and locating only within the open access regions.
```

Comment:
- [ ] maybe remove the word "variants",  (see the paragraph above for Raw SNV Calls)